fe/src/App.css

```
@@ -0,0 +1,8 @@
+.App {
```

다른 클래스명과 같이 케이스를 맞춰주면 좋을 것 같네요~ 
```suggestion
.app {
```